res/skins/Deere/skin.xml

```diff
@@ -17,8 +17,28 @@
       <attribute persist="true" config_key="[Deere],show_parallel_waveforms">0</attribute>
       <attribute persist="true" config_key="[Microphone],show_microphone">0</attribute>
       <attribute persist="true" config_key="[PreviewDeck],show_previewdeck">0</attribute>
-      <attribute config_key="[Library],show_library">1</attribute>
```

Comment:
I have just looked into it, but I could not instantly spot the cause of this issue. 
Would you mind to take care about it. It would be also nice if you double check the other Deere resolved merge conflicts as well. Thank you.